__tests__/Scanner.test.tsx

```diff
@@ -0,0 +1,59 @@
+import { render, screen, act, fireEvent, getByTestId } from "@testing-library/react";
```

Comment:
"act" and "getByTestId" are unused dependencies

Reply:
Uhh make sense, but the testing is pretty much half-baked and it needs a lot of improvement.. I guessed I'll use it later so i just leave it but I could remove it if it bothers